src/storage-proofs/block.ts

```diff
@@ -0,0 +1,36 @@
+import Common, { Chain, Hardfork } from '@ethereumjs/common';
+import { bufferToHex } from 'ethereumjs-util';
```

Comment:
Isn't there a function in ethers.js to do this? It's better reduce the number of dependencies cuz it will be loaded on the frontend.

Reply:
actually can just do natively: replaced with: `const headerRlp = `0x${header.serialize().toString('hex')}`;`